common/compositor/vk/vkshim.h

```diff
@@ -16,6 +16,7 @@
 #define VKSHIM_H_
 
 #include <drm/drm_fourcc.h>
+#include <gbm.h>
```

Comment:
You can add it to platformdefines.h and include that here also I see we are including gbm conversions here we need to figure out a way to isolate them